buildpack_integration_test.go

```diff
@@ -0,0 +1,166 @@
+package acceptance_test
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+
+	"github.com/paketo-buildpacks/occam"
+	. "github.com/paketo-buildpacks/occam/matchers"
+	"github.com/paketo-buildpacks/packit/v2/pexec"
+)
+
+const lifecycleVersion = "0.14.0"
```

Comment:
Instead of hardcoding the lifecycle version can we do something like https://github.com/paketo-buildpacks/jammy-tiny-stack/commit/1eb8ae08421e363586adac2287ab1acf5bb8fbf5

Reply:
Yes, I copied this before we made that change :)